lib/code_corps/github/adapters/github_repo.ex

```diff
@@ -1,22 +1,16 @@
 defmodule CodeCorps.GitHub.Adapters.GithubRepo do
-  def from_api(%{
-    "id" => github_id,
-    "name" => name,
-    "owner" => %{
-      "id" => github_account_id,
-      "avatar_url" => github_account_avatar_url,
-      "login" => github_account_login,
-      "type" => github_account_type
-    }
-  }) do
-    %{
-      github_id: github_id,
-      name: name,
-      github_account_id: github_account_id,
-      github_account_avatar_url: github_account_avatar_url,
-      github_account_login: github_account_login,
-      github_account_type: github_account_type
-    }
+
+  @mapping [
+    {:github_id, ["id"]},
+    {:name, ["name"]},
+    {:github_account_id, ["owner", "id"]},
+    {:github_account_avatar_url, ["owner", "avatar_url"]},
+    {:github_account_login, ["owner", "login"]},
+    {:github_account_type, ["owner", "type"]}
```

Comment:
Just for consistency's sake, would clean up alpha ordering here (and wherever possible).